### PR TITLE
Add user circle position tracking

### PR DESCRIPTION
**Pull request by sztembus**

### CODE REVIEW

**Comment by alvelig** [comment]
Hi, thank yo for the contribution. Why did you close it? 

We would need docs and update EventListener example (https://github.com/react-community/react-native-maps/blob/master/example/examples/EventListener.js) to merge this.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (rborn) [inline .vscode/settings.json]: Please remove the `.vscode` folder

**Comment by alvelig** [comment]
Oops #2032

**Comment by rborn** [comment]
@sztembus @dabit1 we kind of messed up here 😔 it looks like this PR and #2032 are doing the same thing. 

Would be amazing if you guys could have a look and see maybe you can combine the PRs or decide which one to merge?

I'll try to prepare a PR template in order to avoid this kind of situations in the future. @alvelig 

🤗

**Comment by dabit1** [comment]
@rborn My pull request has the same like this but with more data, examples and README edited. If tomorrow I have time I'll try to resolve the conflicts.

**Comment by alvelig** [comment]
@sztembus We appreciate very much your contribution and looking forward to new PRs from new, but we are closing this in favor of #2032, and hope that this fact won't upset you, and we'll see more valuable contributions from you!

**Comment by sztembus** [comment]
Actually, I'm a little upset because my contribution was first, it's also pretty the same as the #2032 (there's only a little more info about location, but that's easy). You guys even changed event name to one that I proposed, that's not fair for sure. There won't be even word about my contribution in this stuff.

**Comment by alvelig** [comment]
That's fair that your contribution was first, but we did not receive any comments from you about that collision that we encountered. Can you please add the data and resolve conflicts and we'll prefer your contribution according to FIFO rule.

**Comment by dabit1** [comment]
Hi guys. I think that here there is a confusion... Just read my first comment on my PR. My PR is actually a copy from #1508 which is older than #2026. I did that because of the author of #1508 doesn't fix the conflicts. So what is not fair...?

**Comment by sztembus** [comment]
Ok, if original author was @sagi and @dabit1 just added some changes then yeah, #2032 is ok for me.

**Comment by alvelig** [comment]
Thanks guys for solving this collision. We are all working for the sake of the community.

**Comment by rborn** [comment]
@sztembus @dabit1 Right now it's only @alvelig and I  (with @alvelig doing most of the work) who are reviewing all the PRs and trying to handle all the issues, so it's not very easy for us. I'm really sorry we got to have this conflict and sometimes I know we might do things that are not fair (each of you spent time in creating the  PR) and can be frustrating, and I'm really thankful you guys managed to handle this situation in a friendly way 🤗

**Comment by alvelig** [comment]
@rborn that's not true, you are the most hard working here XD

**Pull request closed without merging**